publication-commons/src/main/java/no/unit/nva/publication/model/business/FileEntry.java

```diff
@@ -33,39 +39,45 @@ public final class FileEntry implements Entity {
     private final Instant createdDate;
     private Instant modifiedDate;
     private File file;
+    private FileEvent fileEvent;
 
     @JsonCreator
     private FileEntry(@JsonProperty("resourceIdentifier") SortableIdentifier resourceIdentifier,
                       @JsonProperty("createdDate") Instant createdDate,
                       @JsonProperty("modifiedDate") Instant modifiedDate, @JsonProperty("owner") User owner,
                       @JsonProperty("ownerAffiliation") URI ownerAffiliation,
-                      @JsonProperty("customerId") URI customerId, @JsonProperty("file") File file) {
+                      @JsonProperty("customerId") URI customerId, @JsonProperty("file") File file,
```

Comment:
We should probably not have called it "customerId" since "customer" would be better. Oh well.